### PR TITLE
Add DJLServing Java client demo

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

File: djl-serving/java-client/src/main/java/ai/djl/examples/serving/javaclient/HttpUtils.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (frankfliu) [inline djl-serving/java-client/src/main/java/ai/djl/examples/serving/javaclient/HttpUtils.java:1]: Can we upgrade to Java 11 and use Java built-in HttpClient?

File: djl-serving/java-client/gradlew [hunk 2]
```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```
Comment (lanking520) [inline djl-serving/java-client/gradlew:1]: please do not checkin this file

File: djl-serving/java-client/gradlew.bat [hunk 3]
```diff
@@ -0,0 +1,89 @@
+@rem
```
Comment (lanking520) [inline djl-serving/java-client/gradlew.bat:1]: please do not check in this file

**Review by lanking520 — CHANGES_REQUESTED** [review]

File: djl-serving/java-client/src/main/java/ai/djl/examples/serving/javaclient/HttpUtils.java (hunk 1 above)
Comment (xyang16) [reply djl-serving/java-client/src/main/java/ai/djl/examples/serving/javaclient/HttpUtils.java:1]: Done.

File: djl-serving/java-client/gradlew (hunk 2 above)
Comment (xyang16) [reply djl-serving/java-client/gradlew]: OK

File: djl-serving/java-client/gradlew.bat (hunk 3 above)
Comment (xyang16) [reply djl-serving/java-client/gradlew.bat]: OK

**Review by frankfliu — APPROVED** [review]

**Review by lanking520 — APPROVED** [review]

**Pull request merged**